mainsite/package.json

```
@@ -12,12 +12,13 @@
     "url": "git+https://github.com/safetyscore/safetyscore.git"
   },
   "scripts": {
-    "build": "next build",
+    "prepare-docs": "./scripts/prepare-docs.js",
```

This doesn't work on my windows machine, consider the following:
```suggestion
    "prepare-docs": ".\"/scripts/prepare-docs.js\"",
```

---

Thanks, changed